opp_path_selection/include/opp_path_selection/path_selector_parameters.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Southwest Research Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OPP_PATH_SELECTION_PATH_SELECTOR_PARAMETERS_H
+#define OPP_PATH_SELECTION_PATH_SELECTOR_PARAMETERS_H
+
+#include <limits>
```

Comment:
Unnecessary include?

Reply:
Also, you might consider combining this with the path selector file; it doesn't seem necessary to put this in a separate file